### PR TITLE
Suggest using www-data user for behat runs

### LINKED ISSUE #108: behat failures related to touch()
Was performing a complete run locally (docker on MacOSX) and got these 6 consistent failures (rerun them a number of times):

````
001 Scenario: All of the selected roles can award badges # /var/www/html/badges/tests/behat/award_badge.feature:321
      And I trigger cron                                 # /var/www/html/badges/tests/behat/award_badge.feature:389
        Other backtraces found:
        line 93 of /lib/classes/task/file_temp_cleanup_task.php: call to touch()
        line 110 of /lib/classes/task/file_temp_cleanup_task.php: call to core()
        line 189 of /lib/cronlib.php: call to core()
        line 114 of /lib/cronlib.php: call to cron_run_inner_scheduled_task()
        line 73 of /lib/cronlib.php: call to cron_run_scheduled_tasks()
        line 81 of /admin/cron.php: call to cron_run() (Exception)

002 Scenario: Backup and restore of meta enrolment instance # /var/www/html/enrol/meta/tests/behat/enrol_meta.feature:82
      And I trigger cron                                    # /var/www/html/enrol/meta/tests/behat/enrol_meta.feature:100
        Other backtraces found:
        line 93 of /lib/classes/task/file_temp_cleanup_task.php: call to touch()
        line 110 of /lib/classes/task/file_temp_cleanup_task.php: call to core()
        line 189 of /lib/cronlib.php: call to core()
        line 114 of /lib/cronlib.php: call to cron_run_inner_scheduled_task()
        line 73 of /lib/cronlib.php: call to cron_run_scheduled_tasks()
        line 81 of /admin/cron.php: call to cron_run() (Exception)

003 Scenario: Bulk accepting requests # /var/www/html/admin/tool/dataprivacy/tests/behat/manage_data_requests.feature:61
      And I trigger cron              # /var/www/html/admin/tool/dataprivacy/tests/behat/manage_data_requests.feature:88
        Other backtraces found:
        line 93 of /lib/classes/task/file_temp_cleanup_task.php: call to touch()
        line 110 of /lib/classes/task/file_temp_cleanup_task.php: call to core()
        line 189 of /lib/cronlib.php: call to core()
        line 114 of /lib/cronlib.php: call to cron_run_inner_scheduled_task()
        line 73 of /lib/cronlib.php: call to cron_run_scheduled_tasks()
        line 81 of /admin/cron.php: call to cron_run() (Exception)

004 Scenario: Bulk denying requests # /var/www/html/admin/tool/dataprivacy/tests/behat/manage_data_requests.feature:106
      And I trigger cron            # /var/www/html/admin/tool/dataprivacy/tests/behat/manage_data_requests.feature:133
        Other backtraces found:
        line 93 of /lib/classes/task/file_temp_cleanup_task.php: call to touch()
        line 110 of /lib/classes/task/file_temp_cleanup_task.php: call to core()
        line 189 of /lib/cronlib.php: call to core()
        line 114 of /lib/cronlib.php: call to cron_run_inner_scheduled_task()
        line 73 of /lib/cronlib.php: call to cron_run_scheduled_tasks()
        line 81 of /admin/cron.php: call to cron_run() (Exception)

005 Scenario: Receiving notification on site level # /var/www/html/admin/tool/monitor/tests/behat/subscription.feature:101
      And I trigger cron                           # /var/www/html/admin/tool/monitor/tests/behat/subscription.feature:114
        Other backtraces found:
        line 93 of /lib/classes/task/file_temp_cleanup_task.php: call to touch()
        line 110 of /lib/classes/task/file_temp_cleanup_task.php: call to core()
        line 189 of /lib/cronlib.php: call to core()
        line 114 of /lib/cronlib.php: call to cron_run_inner_scheduled_task()
        line 73 of /lib/cronlib.php: call to cron_run_scheduled_tasks()
        line 81 of /admin/cron.php: call to cron_run() (Exception)

006 Scenario: Receiving notification on course level # /var/www/html/admin/tool/monitor/tests/behat/subscription.feature:122
      And I trigger cron                             # /var/www/html/admin/tool/monitor/tests/behat/subscription.feature:135
        Other backtraces found:
        line 93 of /lib/classes/task/file_temp_cleanup_task.php: call to touch()
        line 110 of /lib/classes/task/file_temp_cleanup_task.php: call to core()
        line 189 of /lib/cronlib.php: call to core()
        line 114 of /lib/cronlib.php: call to cron_run_inner_scheduled_task()
        line 73 of /lib/cronlib.php: call to cron_run_scheduled_tasks()
        line 81 of /admin/cron.php: call to cron_run() (Exception)

6 scenarios (6 failed)
261 steps (192 passed, 6 failed, 63 skipped)
6m38.24s (51.80Mb)
````

Worth investigating them, usually touch cries when the user is not exactly the same and things like that...

Ciao :-)

### PR DESCRIPTION
**Pull request by kabalin**
Fixes #108

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,9 @@ Time: 4.45 seconds, Memory: 38.00MB
 OK (2 tests, 7 assertions)
 ```
 
+Notes:
+* If you want to run test with coverage report, use command: `bin/moodle-docker-compose exec webserver phpdbg -qrr vendor/bin/phpunit --coverage-text auth_manual_testcase auth/manual/tests/manual_test.php`
```
Comment (scara) [inline README.md]: Hi @kabalin,
how can you get the code coverage report w/o [e.g.](https://phpunit.readthedocs.io/en/7.5/code-coverage-analysis.html) XDebug extension?
Ref.: https://github.com/moodlehq/moodle-php-apache/issues/17

TIA,
Matteo

Comment (kabalin) [reply README.md]: xdebug is not needed. It works from the box if you execute it as shown :) You might be better off to actually use it on mod_forum to see some sensible output, as it got `coverage.php` defined and relevant annotations.

Comment (scara) [reply README.md]: TNX for your prompt reply: I never tried since PHPUnit docs tells about the need for an extension.

Will do,
Matteo

Comment (kabalin) [reply README.md]: Thechnically you need one of supported debuggers: Xdebug, PCOV or PHPDBG. PHPDBG is php internal one (and in fact the fastest in code coverage tests, google for details).

**Comment by stronk7** [comment]
Sold! Thanks!

**Pull request merged**